otter_welcome_buddy/cogs/english_club_reminder.py

```diff
@@ -0,0 +1,135 @@
+import os
+import re
+
+import discord
+from apscheduler.schedulers.asyncio import AsyncIOScheduler
+from discord import TextChannel
+from discord.ext import commands
+from discord.ext.commands import Bot
+from discord.ext.commands import Context
+
+from otter_welcome_buddy.common.constants import CronExpressions
+from otter_welcome_buddy.common.constants import DiscordCommonRoles
+from otter_welcome_buddy.common.constants import OTTER_HEADER_IMAGE
+from otter_welcome_buddy.common.constants import OTTER_MODERATOR
+from otter_welcome_buddy.common.utils.dates import DateUtils
+from otter_welcome_buddy.common.utils.discord_ import get_channel_by_id
+from otter_welcome_buddy.common.utils.types.common import DiscordChannelType
+from otter_welcome_buddy.formatters import messages
+from otter_welcome_buddy.settings import ENGLISH_CHANNEL_ID
+from otter_welcome_buddy.settings import NON_ENGLISH_TEACHERS_CHANNEL_ID
+
+
+class English(commands.Cog):
+    """
+    English command events, where notifications about English sessions
+    Commands:
+        englishclub start:     Start cronjob for timeline messages
+        englishclub stop:      Stop cronjob for timeline messages
+    """
+
+    def __init__(self, bot: Bot, messages_dependency: type[messages.Formatter]):
+        self.bot: Bot = bot
+        self.messages_formatter: type[messages.Formatter] = messages_dependency
+        self.scheduler: AsyncIOScheduler = AsyncIOScheduler()
+        self.__configure_scheduler()
+
+    @commands.group(
+        brief="Commands related to English Club!",
+        invoke_without_command=True,
+        pass_context=True,
+    )
+    @commands.has_role(OTTER_MODERATOR)
+    async def englishclub(self, ctx: Context) -> None:
+        """
+        English Club will setup an English session
+        """
+        await ctx.send_help(ctx.command)
+
+    @englishclub.command(brief="Start the cronjob for the messages")  # type: ignore
+    @commands.has_role(OTTER_MODERATOR)
```

Comment:
Thinking twice about this, can you also add the admin role just in case?
 ```suggestion
    @commands.has_any_role(OTTER_ADMIN, OTTER_MODERATOR)
```